### PR TITLE
feat: Made Initial component for WelcomeEmail

### LINKED ISSUE #10: Create a Welcome Email
Under the emails folder in components, create one called WelcomeEmail (use this naming convention, not sure why react-email uses dashes...)
Design a nice welcome email similar to the Stripe one that says you've been accepted into program.
Props to this component should be first name and program name.
The button sends you off to "BASE_URL/"
BASE_URL is an environment variable
See the react-email docs or the examples in the repo to see how to design the email

### PR DESCRIPTION
**Pull request by AlanKha**
# Description
Replaced the stripe-welcome.tsx with WelcomeEmail.tsx Basically just took the email from stripe and Changed the wording, links, logo, etc.

Can be tested with the following code in ```src/app/page.tsx```
``` tsx
import React from "react";
import WelcomeEmail from "@/components/emails/WelcomeEmail";

export default function Home() {
  return (
    <>
      <WelcomeEmail firstName="John" programName="Test Program" />
    </>
  );
}

```

I wanted to note that there is a hydration error when testing it this way. I believe this is just an error for how it's being rendered in the browser instead of an email, but I'm not sure.

## Relevant Issues

Closes #10 - Create Welcome Email

### CODE REVIEW

File: src/components/emails/WelcomeEmail.tsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  Html,
+  Link,
+  Preview,
+  Section,
+  Text,
+} from "@react-email/components";
+import * as React from "react";
+import Logo from "@/components/Logo/index";
+
+const baseUrl = process.env.VERCEL_URL
```
Comment (RudraPatel2003) [inline src/components/emails/WelcomeEmail.tsx:16]: Use the BASE_URL environment variable

File: src/components/emails/WelcomeEmail.tsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
+import {
```
Comment (RudraPatel2003) [inline src/components/emails/WelcomeEmail.tsx:1]: Linting rules are now stricter; run `pnpm lint` to fix the import ordering issues

File: src/components/emails/WelcomeEmail.tsx [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+  Text,
+} from "@react-email/components";
+import * as React from "react";
+import Logo from "@/components/Logo/index";
+
+const baseUrl = process.env.VERCEL_URL
+  ? `https://${process.env.VERCEL_URL}`
+  : "";
+
+interface StripeWelcomeEmailProps {
```
Comment (RudraPatel2003) [inline src/components/emails/WelcomeEmail.tsx:20]: 1. Use a type not an interface unless you have a specific reason to use an interface (there is only 1 or 2, which is not seen here)
2. Remove all references to stripe

File: src/components/emails/WelcomeEmail.tsx [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+  firstName: string;
+  programName: string;
+}
+
+export const StripeWelcomeEmail: React.FC<StripeWelcomeEmailProps> = ({
```
Comment (RudraPatel2003) [inline src/components/emails/WelcomeEmail.tsx:25]: Remove references to Stripe

Also I would prefer to keep the components consistent in how they type stuff. so something like

```
export default function WelcomeEmail({name, programName}: WelcomeEmailProps) {...}

File: src/components/emails/WelcomeEmail.tsx [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+}) => (
+  <Html>
+    <Head />
+    <Preview>
+      You&apos;ve been accepted into St. Christopher&apos;s Trucker Relief Fund!
+    </Preview>
+    <Body style={main}>
+      <Container style={container}>
+        <Section style={box}>
+          <Logo
```
Comment (RudraPatel2003) [inline src/components/emails/WelcomeEmail.tsx:37]: Right now, your email doesn't build if you use `pnpm run email-dev`
According to react-email docs, SVGs are not well supported in email clients, which is what our logo component uses. My bad on that one, but I added a logo.png to the repo. Reference that and use the react-email image component instead (unless Next.js starts complaining and wants you to use theirs)

File: src/components/emails/WelcomeEmail.tsx [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+          <Text style={paragraph}>
+            If you have any further questions or concerns, please don&apos;t
+            hesitate to reach out to us at our{" "}
+            <Link style={anchor} href="https://truckersfund.org/contact-us">
+              contact page.
+            </Link>
+          </Text>
+          <Hr style={hr} />
+          <Text style={footer}>
+            St. Christopher&apos;s Trucker Relief Fund, Phone: (865) 202 - 9428
```
Comment (RudraPatel2003) [inline src/components/emails/WelcomeEmail.tsx:61]: Would prefer it to be (865) 202-9428
The spacing seemed weird on the email when running dev

File: src/components/emails/WelcomeEmail.tsx [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+          </Text>
+        </Section>
+      </Container>
+    </Body>
+  </Html>
+);
+
+export default StripeWelcomeEmail;
```
Comment (RudraPatel2003) [inline src/components/emails/WelcomeEmail.tsx:69]: Remove references to stripe

File: src/components/emails/stripe-welcome.tsx
Comment (RudraPatel2003) [inline src/components/emails/stripe-welcome.tsx:1]: I say keep all the example emails for now. Could you bring this back?

**Review by RudraPatel2003 — CHANGES_REQUESTED** [review]
Nice work, just gotta polish it up a bit to keep our repo consistent and fix that rendering issue

Make sure you have the ESLint VSCode extension installed to fix all warnings and errors in the future!

Merge with main to get the PNG logo (which will also make certain linting rules fail the build)

**Comment by AlanKha** [comment]
Everything looks good on my end

**Review by RudraPatel2003 — APPROVED** [review]

**Pull request merged**